rules/web/proxy_generic/proxy_download_susp_tlds_blacklist.yml

```
@@ -12,7 +12,7 @@ references:
     - https://krebsonsecurity.com/2018/06/bad-men-at-work-please-dont-click/
 author: Florian Roth (Nextron Systems)
 date: 2017/11/07
-modified: 2023/01/09
+modified: 2023/05/18
```

No changes were made, so no need for this.

```suggestion
modified: 2023/01/09
```